spec/requests/posts_spec.rb

```diff
@@ -10,9 +10,12 @@
     end
   end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍